files/es/learn/css/first_steps/how_css_is_structured/index.html

```diff
@@ -509,5 +509,5 @@ <h2 id="En_este_modulo">En este modulo</h2>
  <li><a href="/en-US/docs/Learn/CSS/First_steps/Getting_started">Empezar con el CSS</a></li>
  <li><a href="/en-US/docs/Learn/CSS/First_steps/How_CSS_is_structured">Cómo se estructura el CSS</a></li>
  <li><a href="/en-US/docs/Learn/CSS/First_steps/How_CSS_works">Cómo funciona CSS</a></li>
```

Comment:
Las url se ponen hacia `es` para que, ya sea exista o no el bot lo haga automáticamente 

```suggestion
 <li><a href="/es/docs/Learn/CSS/First_steps/Getting_started">Empezar con el CSS</a></li>
 <li><a href="/es/docs/Learn/CSS/First_steps/How_CSS_is_structured">Cómo se estructura el CSS</a></li>
 <li><a href="/es/docs/Learn/CSS/First_steps/How_CSS_works">Cómo funciona CSS</a></li>
```